gunicorn/app/wsgiapp.py

```diff
@@ -31,11 +31,15 @@ def init(self, parser, opts, args):
             from .pasterapp import paste_config
             return paste_config(self.cfg, self.cfgurl, self.relpath)
 
-        if len(args) < 1:
+        if len(args):
```

Comment:
You should set directly  `self.app.uri` insteadof introducing another variable. Also please check `if len(args) > 0` so it's more clear about what it does.